core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -190,6 +193,11 @@
             CSV_SPLITTER.splitToList(getNormalized(
                     "cachedHeaders", DEFAULT_CACHED_HEADERS, CharMatcher.ascii()::matchesAllOf));
 
+    private static final String DEFAULT_ANNOTATED_SERVICE_EXCEPTION_VERBOSITY = "unhandled";
```

Comment:
`ExceptionVerbosity.UNHADLED.name()`?

Reply:
It makes `{@value ...}` not work. So I intentionally used the string here. :-)